sources/packages/web/src/views/aest/institution/ApproveDenyDesignation.vue

```diff
@@ -1,33 +1,75 @@
 <template>
-  <!-- CSS class temporary-modal is work around for lack of responsiveness of v-dialog. -->
-  <modal-dialog-base
-    :showDialog="showDialog"
-    @dialogClosed="dialogClosed"
-    :title="title"
-  >
-    <template v-slot:content>
-      <div class="temporary-modal">
-        <formio
-          formName="approvedenydesignation"
-          :data="designation"
-          @loaded="formLoaded"
-          @submitted="submitDesignationUpdate"
-        ></formio>
-      </div>
-    </template>
-    <template v-slot:footer>
-      <check-permission-role :role="Role.InstitutionApproveDeclineDesignation">
-        <template #="{ notAllowed }">
+  <v-form ref="approveDenyDesignation">
+    <modal-dialog-base :showDialog="showDialog" :title="title" max-width="730">
```

Comment:
`:max-width?`

Reply:
I mentioned in previous PRs, it seems that we have been adding `max-width="730"` in absolute all models, why not set this as a default value in the component and stop doing it?

Reply:
I'll default it in this change, and will remove the changed components in the next PR